docs/en/serial-vault.md

```diff
@@ -0,0 +1,415 @@
+That snap includes a [prepare-device][prepare-device] hook script where
+details of the system and Serial Vault connection such as the URL, API key,
+and serial number are defined.
+
+The following table explains the configuration settings that need to be in the
+prepare-device hook:
+
+| snapd setting                | Description                                                                                                                                                                                                                                                                                                                                                                                                                            |   |   |   |
+|------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---|---|---|
+| device-service.url           | The base API URl for the Serial Vault, ending in a trailing slash ("/"). <br> This value must be set to: `https://serial-vault-partners.canonical.com/v1`                                                                                                                                                                                                                                                |   |   |   |
+| device-service.headers       | The API key defined as an HTTP header in JSON string format.  This value must match the one configured for this device model in the SerialVault Admin portal web user interface. <br> You can generate a valid API key with `$pwgen 64 1` and then copy it into the Serial Vault when creating your Model. Or, you can auto create the API key in the Serial Vault when creating a Model and copy it into the `prepare-device` hook script. |   |   |   |
+| registration.proposed-serial | The unique serial number of the device. How to get this serial depends on the device platform.                                                                                                                                                                                                                                                                                                                                         |   |   |   |
+
+## Getting the account-id of your Brand SSO account
+
+Requirements:
+
+ - snapd must be installed.
+ - snapcraft must be installed.
+ - Must be snapcraft logged on as Brand SSO account.
+
+You need to generate a key used by the Serial Vault to sign serial assertions
+and system-user assertions. These keys must be generated under the Brand SSO
```

Comment:
Link for system-user assertion: https://docs.ubuntu.com/core/en/reference/assertions/system-user